src/coreComponents/linearAlgebra/interfaces/hypre/HyprePreconditioner.cpp

```diff
@@ -74,6 +74,11 @@ HyprePreconditioner::HyprePreconditioner( LinearSolverParameters params,
         createILUT();
         break;
       }
+      case LinearSolverParameters::PreconditionerType::direct:
+      {
+        createDirect();
```

Comment:
Does this mean that a direct solver can be used as a preconditioner, like in a block preconditioner?

Reply:
Yeah, that's the goal. Mainly for debugging purposes, when block preconditioner is not performing well.

Reply:
Nice! It will be useful!